From: Jonathan Carter <jcc@debian.org>
Date: Tue, 16 Oct 2018 10:49:02 +0200
Subject: use changelog commit time for build time

Use the package commit time as a release time stamp
instead of the build time in order to make the build
reproducible.
---
 Makefile | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index afaea32..4de5533 100644
--- a/Makefile
+++ b/Makefile
@@ -10,7 +10,13 @@
 # The version number, for release:
 
 VER_VERSION:=0.9.22
-VER_DATE:=$(shell date +"%Y-%m-%d")
+# Original upstream date generation:
+#VER_DATE:=$(shell date +"%Y-%m-%d")
+# In Debian, we use the package commit date
+# so that the package can be reproducible
+PKG_DATE:=%(shell grep "\-\-" debian/changelog | head -n 1 | cut -d ">" -f 2 | cut -b 3-)
+VER_DATE:=%(shell date +"%Y-%m-%d" -d $(PKG_DATE))
+
 MAGIC_API_VERSION:=0x00000003
 
 # Need to know the OS
